### PR TITLE
New Navy Ships

### PR DESCRIPTION
**Pull request by Brick63**
**Content**
Introducing the logistical marvels that are the Auxiliary and the Dropship.

No longer do you have to take powerful warships away from the front lines to move around supplies or people, with its innovative modular construction, the Auxiliary can now handle all those tasks while the warships go and fight whatever people are afraid of currently.

But lets not forget its little friend, the Dropship allows you to move soldiers into dangerous areas without risking the large, expensive transport in the process.

Assets:
[auxiliary assets.zip](https://github.com/endless-sky/endless-sky/files/4679342/auxiliary.assets.zip)

Repost of #4004 because of an issue with the test file not updating

High dpi: https://github.com/endless-sky/endless-sky-high-dpi/pull/59

### CODE REVIEW

File: data/human/ships.txt [hunk 1]
```diff
@@ -1202,6 +1307,51 @@ ship "Dreadnought"
...
+		"X1200 Ion Steering"
+	
+	engine 13 26
+	engine -13 26
+	gun 2 -27 "Beam Laser"
+	gun -2 -27 "Beam Laser"
+	leak "leak" 60 50
+	explode "tiny explosion" 15
+	explode "small explosion" 5
+	description "Dropships were designed with the Navy's Auxiliary in mind. Auxiliaries often act as a forward operating base when landed on planets, so moving the entire Auxiliary isn't much of an option when a large number of troops need to be moved to a different sector. That's where the Dropship comes in, a small, fast, and cramped vessel able to move considerable numbers of troops at great speeds for its size."
```
Comment (Pointedstick) [inline data/human/ships.txt:1352]:
```suggestion
	description "Dropships were designed with the Navy's Auxiliary in mind. Landed Auxiliaries often act as forward operating bases, so troops needing transport to another sector rely on their Dropships. This cramped and uncomfortable vessel is able to move a squad number of Republic soldiers at great speed for its size."
```

File: data/human/ships.txt [hunk 2]
```diff
@@ -1202,6 +1307,51 @@ ship "Dreadnought"
...
+		"X1200 Ion Steering"
+	
+	engine 13 26
+	engine -13 26
+	gun 2 -27 "Beam Laser"
+	gun -2 -27 "Beam Laser"
+	leak "leak" 60 50
+	explode "tiny explosion" 15
+	explode "small explosion" 5
+	description "Dropships were designed with the Navy's Auxiliary in mind. Landed Auxiliaries often act as forward operating bases, so troops needing transport to another sector rely on their Dropships. This cramped and uncomfortable vessel is able to move a squad number of Republic soldiers at great speed for its size."
```
Comment (Pointedstick) [inline data/human/ships.txt:1352]:
```suggestion
	description "Dropships were designed with the Navy's Auxiliary in mind. Landed Auxiliaries often act as forward operating bases, so troops needing transport to another sector rely on their Dropships. This cramped and uncomfortable vessel is able to move a squad of Republic soldiers at great speed for its size."
```

If I was the one who suggested this typo, I apologize. :)

File: data/human/ships.txt [hunk 3]
```diff
@@ -183,6 +183,111 @@ ship "Arrow"
...
+	leak "leak" 30 50
+	leak "flame" 30 80
+	leak "big leak" 30 30
+	explode "tiny explosion" 20
+	explode "small explosion" 45
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 10
+	"final explode" "final explosion large"
+	description "The Auxiliary is the Republic Navy's logistical backbone. The size and durability of a capital ship but more cargo and passenger space than any other Navy ship, the Auxiliary is used whenever the Navy needs large amounts of troops or cargo transported with a guarantee that they'll reach the destination."
```
Comment (TheMarksman-ES) [inline data/human/ships.txt:252]: description "The Auxiliary is, by all accounts, the Republic Navy's logistical backbone. With the size and durability of a capital ship but more cargo and passenger space than any other Navy ship, the Auxiliary is used whenever the Navy requires large amounts of troops or cargo transported with a guarantee that they'll reach the destination."

File: data/human/ships.txt [hunk 4]
```diff
@@ -183,6 +183,111 @@ ship "Arrow"
...
+	description "	The Auxiliary was built with modularity in mind, allowing the ship to be fitted for more cargo or more bunk space depending on the mission, but this modularity makes the ship rather expensive to build, and therefore a rare sight to see."
```
Comment (TheMarksman-ES) [inline data/human/ships.txt:253]: description "	The Auxiliary was built with modularity in mind, allowing the ship to be fitted for more cargo or bunk space depending on the mission. This modularity makes the ship rather expensive to build, however, and therefore a rare sight to see."

File: data/human/ships.txt [hunk 5]
```diff
@@ -1202,6 +1307,51 @@ ship "Dreadnought"
...
+		"X1200 Ion Steering"
+	
+	engine 13 26
+	engine -13 26
+	gun 2 -27 "Beam Laser"
+	gun -2 -27 "Beam Laser"
+	leak "leak" 60 50
+	explode "tiny explosion" 15
+	explode "small explosion" 5
+	description "Dropships were designed with the Navy's Auxiliary in mind. Landed Auxiliaries often act as forward operating bases, so troops needing transport to another sector rely on their Dropships. This cramped and uncomfortable vessel is able to move a squad of Republic soldiers at great speed for its size."
```
Comment (TheMarksman-ES) [inline data/human/ships.txt:1352]: description "Dropships were designed with the Navy's Auxiliary in mind. Landed Auxiliaries often act as forward operating bases, so troops needing transport to another area often rely on their Dropships. This cramped and uncomfortable vessel is able to move a squad of Republic soldiers at great speed for its size."

**Pull request merged**